src/autorange.jl

```diff
@@ -0,0 +1,54 @@
+function autorange(img, tform)
+    R = CartesianRange(indices(img))
+    I = first(R)
+    mn = mx = x = tform(SVector(I))
+    for I in CornerIterator(R)
+        x = tform(SVector(I))
+        # we map min and max to prevent type-inference issues
+        # (because min(::SVector,::SVector) -> Vector)
+        mn = map(min, x, mn)
+        mx = map(max, x, mx)
```

Comment:
I changed this code because of type inference issues. before it was `mn = min(x,mn)` etc